descriptions.md

```diff
@@ -32,6 +31,15 @@ A client that offers a simple stateless API to interact directly with Ably's RES
 ||| `PaginatedResult` || A [`PaginatedResult`]{@link PaginatedResult} object containing an array of [`Stats`]{@link Stats} objects. |
 | time() => io Time ||| RSC16 | Retrieves the time from the Ably service as milliseconds since the Unix epoch. Clients that do not have access to a sufficiently well maintained time source and wish to issue Ably [`TokenRequest`s]{@link TokenRequest} with a more accurate timestamp should use the [`queryTime`]{@link ClientOptions#queryTime} property instead of this method. |
 ||| `Time` || The time as milliseconds since the Unix epoch. |
+| batchPublish(BatchPublishSpec) => io BatchResult<BatchPublishSuccessResult \| BatchPublishFailureResult> ||| RSC22 | Publishes a [`BatchPublishSpec`]{@link BatchPublishSpec} object to one or more channels, up to a maximum of 100 channels. |
```

Comment:
idk if this change is for the best - in most of these rows there are no backticks (although there are some dotted around). it would be nice to be consistent but leaving them with no backticks is more aligned with the existing entries.

Reply:
oh, i just realised that backticks are only used for generics types for some reason. not sure why we went with that but i'll update 🙂